client/dashboard/sites/overview-card/style.scss

```diff
@@ -68,3 +68,14 @@
 .dashboard-overview-card__progress-bar {
 	width: 100% !important;
 }
+
+.dashboard-overview-card-upsell-with-action {
+	.components-card__body {
+		border-radius: 8px !important; /* stylelint-disable-line scales/radii */
+		overflow: hidden;
+	}
+
+	.components-button svg path {
+		fill: var( --wp-admin-theme-color );
+	}
+}
```

Comment:
we need newline here

Reply:
Added in ce39da91b38be985610df1a1454f568fbac27fec